src/Header/Hamburger/HamMenu.js

```diff
@@ -6,10 +6,10 @@ export default function HamMenu({ isOpen, closeMenu, scroll }) {
     return(
             <StyledMenu open = {isOpen} scroll = {scroll}>
                 <Link to = "/" onClick = {() => closeMenu()}>Home</Link>
-                <Link to = "/menu" onClick = {() => closeMenu()}>Menu</Link>
-                <Link to = "/about" onClick = {() => closeMenu()}>About</Link>
-                <Link to = "/contact" onClick = {() => closeMenu()}>Contact Us</Link>
-                <Link to = "/testimonies" onClick = {() => closeMenu()}>Testimonies</Link>
+                <Link to = "/Menu" onClick = {() => closeMenu()}>Menu</Link>
```

Comment:
You actually had it right the first time -- links aren't usually capitalized.

Reply:
fixed